shared/haml/user_header.haml

```diff
@@ -83,9 +83,14 @@
         %a.linktag#user-edit{href: CDO.studio_url('users/edit')}= I18n.t("#{loc_prefix}settings")
         %a.linktag#user-signout{href: CDO.studio_url('users/sign_out')}= I18n.t("#{loc_prefix}logout")
 - else
-  %a.linktag{href: CDO.studio_url('users/sign_in'), class: 'button-signin', id: 'signin_button'}
-    .header_button.header_user
-      %span= I18n.t("#{loc_prefix}signin")
+  - if show_updated_header
+    %a.linktag{href: CDO.studio_url('users/sign_in'), class: 'button-signin', id: 'new_signin_button'}
+      .header_button.header_user
+        %span= I18n.t("#{loc_prefix}signin")
+  - else
+    %a.linktag{href: CDO.studio_url('users/sign_in'), class: 'button-signin', id: 'signin_button'}
+      .header_button.header_user
+        %span= I18n.t("#{loc_prefix}signin")
```

Comment:
I don't like the duplication here, but it was the safest way to selectively style the new header without affecting the legacy button.

Reply:
Seems okay since this is temporary work and having both before & after work properly is more important than code hygiene for this temporary state, I think.